File: basic-processor-sample/src/main/java/me/strlght/basic_processor_sample/MainActivity.java
package me.strlght.basic_processor_sample;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.RelativeLayout;
import android.widget.TextView;

import me.strlght.BindView;

public class MainActivity extends AppCompatActivity {
    @BindView(R.id.activity_main)
    RelativeLayout rootLayout;

    @BindView(R.id.sample_text)
    TextView sampleText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        MainActivity_ViewBinding.bind(this);
        sampleText.setText("Hello annotations!");
    }
}

File: basic-kotlin-sample/src/main/kotlin/me/strlght/basic_kotlin_sample/MainActivity.kt
package me.strlght.basic_kotlin_sample

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.RelativeLayout
import android.widget.TextView
import me.strlght.BindView

class MainActivity : AppCompatActivity() {
    @BindView(R.id.activity_main)
    lateinit var rootLayout: RelativeLayout

    @BindView(R.id.sample_text)
    lateinit var sampleText: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        MainActivity_ViewBinding.bind(this)
        sampleText.text = "Hello annotations!"
    }
}
